java/src/jmri/jmrix/cmri/serial/cmrinetmanager/CMRInetManagerFrame.java

```diff
@@ -74,7 +74,7 @@ public void initComponents() {
 	    initializeNodes();
 
         // set the frame's initial state
-        setTitle(Bundle.getMessage("WindowTitle"));
+        setTitle(Bundle.getMessage("WindowTitle") + " - Connection "+_memo.getUserName());
```

Comment:
No need to change this, just a suggestion for later code, but you can make it a bit easier for the I18N crew by defining a new text string in the Bundle and have it handle the merge, similar to what you did elsewhere with the "StatusLine3" constant.

Reply:
Will do this on the next release for all of the localization pieces.